hapi/src/utils/lifebankcode.js

```diff
@@ -60,7 +60,7 @@ const addLifebank = (
 const upLifebank = (
   account,
   password,
-  { name, has_immunity_test, geolocation, ...profile }
+  { name, hasImmunityTest, geolocation, ...profile }
```

Comment:
same problem here, the FE will send `has_immunity_test` so you should use `has_immunity_test: hasImmunityTest `